### ISSUE TITLE
Capability to determine if vlan requirement during launch is datacenter constrained.

### ISSUE THREAD

**Issue by binod33** [dasein-cloud/dasein-cloud-core#14]
Certain clouds like vCloud, requires that a server launched in a network be in the same zone of the network specified. So a capability like isVlanRequirementDcConstrained might be helpful.

**Comment by drewlyall** [comment]
Isn't that just the same as asking whether the VLAN is datacenter constrained?
We already have a capability for that. Unless I'm missing a point of the vCloud case I think it's enough to check that capability? Let me know if you think not.

**Comment by binod33** [comment]
hmm I think i made a mistake describing the method for this issue.
I was thinking more like isVmLaunchNetworkDcConstrained.

**Comment by drewlyall** [comment]
Yeah, I think I see what you're looking for but I think as long as the vlan is either region or dc constrained, that will be how it is for the VM launch. I suspect it might be sufficient to check that capability?
